core/payment/src/accounts.rs

```diff
@@ -0,0 +1,80 @@
+use lazy_static::lazy_static;
+use serde::{Deserialize, Serialize};
+use std::env;
+use std::path::Path;
+use tokio::fs;
+use ya_core_model::driver::{driver_bus_id, AccountMode, Init};
+use ya_core_model::identity;
+use ya_service_bus::typed as bus;
+
+pub const DEFAULT_PAYMENT_DRIVER: &str = "gnt";
+
+lazy_static! {
+    pub static ref ACCOUNT_LIST_PATH: String =
+        env::var("ACCOUNT_LIST").unwrap_or("accounts.json".to_string());
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub(crate) struct Account {
+    pub driver: String,
+    pub address: String,
+    pub send: bool,
+    pub receive: bool,
+}
+
+pub(crate) async fn init_account(account: Account) -> anyhow::Result<()> {
+    log::debug!("Initializing payment account {:?}...", account);
+    let mut mode = AccountMode::NONE;
+    mode.set(AccountMode::SEND, account.send);
+    mode.set(AccountMode::RECV, account.receive);
+    bus::service(driver_bus_id(account.driver))
+        .call(Init::new(account.address, mode))
+        .await??;
+    log::debug!("Account initialized.");
```

Comment:
i think this should also override / extend the file account json file

Reply:
You mean to persist all accounts initialized from CLI?

Reply:
yes, these are the scenarios i was thinking about
- Default id got updated, payment init called with new default address ( should override )
- Calling payment init -r on default identiry ( should update the line in json )
- payment init called on other identity ( should extend the json )

i'm just thinking out loud, we can make issues from scenarios that are too large to be picked up in this task

Reply:
These look out of the scope for this task. It was just supposed to initialize accounts from file. I threw in creating a file with the default identity as a bonus because it makes the default case easier. We can discuss improvement ideas in the next sprint.